nemo_asr/run_fast_conformer_rnnt.sh

```diff
@@ -3,8 +3,8 @@
 export PYTHONPATH="..":$PYTHONPATH
 
 #considering FC-L, FC-XL, FC-XXL, C-L and C-S RNNT models
-MODEL_IDs=("nvidia/parakeet-rnnt-1.1b" "nvidia/parakeet-rnnt-0.6b" "nvidia/stt_en_fastconformer_transducer_large" "nvidia/stt_en_fastconformer_transducer_xlarge" "nvidia/stt_en_fastconformer_transducer_xxlarge" "nvidia/stt_en_conformer_transducer_large" "stt_en_conformer_transducer_small")
-BATCH_SIZE=8
+MODEL_IDs=("nvidia/parakeet-tdt-1.1b" "nvidia/parakeet-rnnt-1.1b" "nvidia/parakeet-rnnt-0.6b" "nvidia/stt_en_fastconformer_transducer_large" "nvidia/stt_en_conformer_transducer_large" "stt_en_conformer_transducer_small")
+BATCH_SIZE=64
```

Comment:
Are you happy fixing your bsz for all models? Or want to sweep over bsz and set the max possible bsz for each model (e.g. as per [this comment](https://github.com/huggingface/open_asr_leaderboard/pull/20#discussion_r1482174219))? You're free to do this under the new leaderboard constraints, and will probably squeeze out some extra perf for the smaller models.

Note: because we've sorted the dataset longest -> shortest, you should get an OOM on the first batch. This should help in sweeping over bsz's.

Reply:
Unfortunately large batch size doesn;t exactly correspond to higher RTFx due to time taken per batch, and we ran some ablations for couple of models but not for smaller models.